addOns/network/src/main/javahelp/help/contents/ratelimit.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<HTML>
+<HEAD>
+<TITLE>
+Rate Limit
+</TITLE>
+</HEAD>
+<BODY>
+<H1>Rate Limit</H1>
+The Rate Limit add-on limits the request rate to hosts or domains to prevent overloading the target or being blocked.
+The limit is applied using a matcher against the request host to pick an individual host, an entire domain, or anything
+between. In the case a host matches multiple rules, the rule with the most restrictive rate is used.
+<p>
+It is accessible via the Options menu and right click menu. Right click on a message to add a rule to limit by the host
```

Comment:
"right click menu" → "context menu", the context menu can be accessed through several ways not just right click.